packages/cfd/src/Stores/Modules/CFD/Helpers/cfd-config.ts

```diff
@@ -113,6 +139,12 @@ export const getMtCompanies = (is_eu: boolean) => {
                 title: localize('Demo'),
                 short_title: all_config.short_title,
             },
+            ctrader: {
+                mt5_account_type: all_config.account_type,
+                leverage: '500',
+                title: localize('Demo'),
+                short_title: localize('cTrader'),
+            },
```

Comment:
should we add a new const `getCtraderCompanies`  and move the cTrader config in that as we are doing for DerivEZ